serverless/event_sources/knative-event-sources.adoc

```diff
@@ -0,0 +1,31 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Getting started with event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+toc::[]
+
+An _event source_ is a resource object, which is the link between an event producer and a _sink_ (or consumer) that receives those events. Currently, {ServerlessProductName} supports the following event source types:
+
+ApiServerSource:: Connects a sink to the Kubernetes API server.
+PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
+
+SinkBinding is also supported, which allows you to connect core Kubernetes resources such as `Deployment`, `Job`, or `StatefulSet` with a sink.
+
+You can create and manage Knative event sources using the Developer perspective, the `kn` CLI, or by applying YAML files.
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* You must create the `default` broker in the same namespace that the event source will be installed in. For more information, see xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing].
```

Comment:
that's not correct.

The Sources do NOT require a broker to be installed. Any compliant source can sink to:
* Knative Serving Services
* Knative Eventing Channel
* Knative Eventing Broker

More than that, they can emit their events to:
* standard k8s Services
* URI (including off cluster urls)

Reply:
Thanks Matthias - I'm trying to get this structured right across our event source docs, so I appreciate the feedback.